gammapy/stats/feldman_cousins.py

```diff
@@ -380,6 +381,9 @@ def fc_find_average_upper_limit(x_bins, matrix, upper_limit, mu_bins):
         Feldman Cousins upper limit x-coordinates
     mu_bins : array-like
         The bins used in mue direction.
+    prob_limit : double
```

Comment:
Funny that you notice this now, I have used double everywhere in the documentation so far ;)

Changed.